physical/consul.go

```diff
@@ -154,6 +154,10 @@ func newConsulBackend(conf map[string]string, logger log.Logger) (Backend, error
 
 	// Configure the client
 	consulConf := api.DefaultConfig()
+	// Set MaxIdleConnsPerHost to the number of processes used in expiration.Restore
+	tr := cleanhttp.DefaultPooledTransport()
+	tr.MaxIdleConnsPerHost = 64
```

Comment:
I've run into this problem and currently have been putting values into `logical` but doesn't belong there.

The right answer might be to create a `helper/const` package that literally just holds const values (that should be exported/shared). Feel free to do that here and I'll port some things over...